config/custom_items/clothing/toggleable_glasses/kozak411-matvisor.json

```
@@ -0,0 +1,9 @@
+{
+  "ckey" :            "kozak411",
+  "character_name" :  "Kaitlyn Whirlow",
+  "item_name" :       "Material Visor",
+  "item_desc" :       "A slim, orange visor. A confusing HUD is displayed on the lenses.",
+  "item_icon_state" : "mat_visor",
+  "item_path" :       "/obj/item/clothing/glasses/material",
+  "additional_data" : { "icon_on" : "mat_visor", "icon_off" : "mat_visor_off"}
```

Material goggles are normally restricted to the Chief Engineer, Senior Engineer, Engineer, and Miner roles - Since I know this character also spawns as a passenger, please either get permission to spawn with these as other roles or add the list of title restrictions:
```suggestion
  "additional_data" : { "icon_on" : "mat_visor", "icon_off" : "mat_visor_off"},
  "req_titles": ["Chief Engineer", "Senior Engineer", "Engineer", "Prospector"]
```